storybook/tailwind.config.js

```diff
@@ -116,7 +116,8 @@ module.exports = {
 		  xxl: '2.5rem',
 		  '3xl': '3rem',
 		  '4xl': '4rem',
-		  '5xl': '8rem'
+		  '5xl': '8rem',
+		  '16xl': '16rem'
```

Comment:
please explain the changes made into configs in your PR description.

Reply:
the same 
404 not found i think you need to get latest version

Reply:
this is the most recent version on your GitHub branch

Reply:
![New Bitmap Image](https://user-images.githubusercontent.com/68795300/107866679-56724580-6e84-11eb-942c-7ef64ee21739.jpg)